src/components/SettingsDialog/ProfileSettings/ProfileSettings.tsx

```diff
@@ -56,6 +61,14 @@ export const ProfileSettings = () => {
           </div>
         </div>
       </div>
+      <Footer visible={isFooterVisible} className="profile-settings__footer">
```

Comment:
It's better to hide a component by doing this: `{isFooterVisible && (<Footer>...</Footer>)}`. This way, the component is prevented from having a lifecycle even tho it's hidden.

Reply:
Thanks for the advice :)